src/components/LeftNav.tsx

```diff
@@ -49,7 +49,7 @@ const subheadingStyle = {
   display: "flex",
   alignItems: "center",
 };
-const activeStyle = {
+const activestyle = {
```

Comment:
Please change back to camelCase - JavaScript objects in general need to be camel case.

Reply:
Warning: React does not recognize the `activeStyle` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `activestyle` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

Thats why I changed it, but I see now that it wasnt talking about the object.